src/schema/v1/artwork/artworkContextGrids/ArtistArtworkGrid.ts

```diff
@@ -46,12 +46,13 @@ export const ArtistArtworkGridType = new GraphQLObjectType<
           artwork,
         })
         gravityArgs.sort = "-published_at"
+        gravityArgs.filter = ["for_sale"]
```

Comment:
Does the existing `for_sale` property on `gravityArgs` work in this case or do we need this new `filter` attribute?

```suggestion
        gravityArgs.for_sale = true
```

Reply:
We can pass only [filter param](https://github.com/artsy/gravity/blob/master/app/api/v1/artists_artworks_endpoint.rb#L14) for this ArtistArtworks endpoint